package.json

```diff
@@ -3,14 +3,19 @@
   "version": "4.2.0",
   "description": "FantasyLand compliant (monadic) alternative to Promises",
   "main": "fluture.js",
+  "files": [
+    "es5.js"
+  ],
```

Comment:
`fluture.js` is automatically published as it is the `main` entry point - so there's no need :)